Watchman.Engine/Generation/Generic/CloudWatchCloudFormationMultiStackTemplate.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json.Linq;
+using Watchman.Configuration;
+
+namespace Watchman.Engine.Generation.Generic
+{
+    public class CloudWatchCloudFormationMultiStackTemplate
```

Comment:
It feels a bit like the multiple-stack logic is spread across this class and `CloudFormationAlarmCreator`
I wonder if it might be better to have a different implementation of `IAlarmCreator` and try to pull any shared logic out elsewhere

Reply:
although actually maybe we don't even need the old implementation